src/Symfony/Component/Config/Tests/Util/XmlUtilsTest.php

```diff
@@ -177,6 +177,11 @@ public function testLoadWrongEmptyXMLWithErrorHandler()
             error_reporting($errorReporting);
 
             throw $e;
+        } catch (\Throwable $e) {
```

Comment:
Yep, but the test case works and no fatal error is expected. If one arise, the test suite will fail anyway... Let's keep things simple IMHO.